nilearn/glm/tests/test_second_level.py

```diff
@@ -138,6 +138,12 @@ def test_check_second_level_input():
     with pytest.raises(ValueError,
                        match="second_level_input must be a list"):
         _check_second_level_input(1, None, flm_object=False)
+    with pytest.raises(ValueError, 
+                       match="Contradictory arguments: flm_object is set to "
+                             "False yet second_level_input is a list of "
+                             "FirstLevelModel objects"):
```

Comment:
In order to pass this test, this needs to match either exactly the error text raised including the spaces (in the implementation you don't include the spaces at the end of each string portion like you do here) or part of the string (e.g. `match="Contradictory arguments: flm_object is set to"):`

Reply:
Alright ! Thanks, i'll make sure the text is identical :)